<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonseeTest</class>
 <widget class="QWidget" name="DonseeTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>设置密码</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>取卡号</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>扇区号</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>数据块</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>重置为初始密码</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>鸣响100ms</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>读</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="3">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>写</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>数据</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
